src/zappy_gui_src/Core/SFML/SFML.cpp

```diff
@@ -19,7 +19,7 @@ SFML::SFML() : _run(true)
     _window.get()->setFramerateLimit(60);
     _map.setWindow(_window);
     _map.setEvent(_event);
-    _map.setMapSize(sf::Vector2f(20, 10));
+    _map.setMapSize(sf::Vector2f(1000, 500));
```

Comment:
Better way is to use static const int because i don't even know what 1000 is supposed to be